src/index.tsx

```diff
@@ -1,12 +1,52 @@
-import ReactDOM from 'react-dom/client';
-import { BrowserRouter as Router } from 'react-router-dom';
-import App from './App';
+import {
+  InstallPWAContextProvider,
+  isIosOrSafariDesktop,
+} from "@adorsys-gis/usepwa";
+import ReactDOM from "react-dom/client";
+import { BrowserRouter as Router } from "react-router-dom";
+import App from "./App";
 
 const root = ReactDOM.createRoot(
-  document.getElementById('root') as HTMLElement
+  document.getElementById("root") as HTMLElement
 );
-root.render(
-  <Router>
-    <App />
-  </Router>
-);
\ No newline at end of file
+
+function TempApp() {
+  return (
+    <Router>
+      {/* Wrap App with InstallPWAContextProvider */}
+      <InstallPWAContextProvider
+        component={isIosOrSafariDesktop() ? "tooltip" : "banner"}
+        installPromptTimeout={30000}
+      >
```

Comment:
I'm not prompted to install the PWA. Can you debug my environment whenever possible?